app.js

```diff
@@ -11,7 +11,7 @@ server.addModel(Arrow.Model.extend('tbray', {
 		// TODO: Add fields to it.
 		title: {type: String}
```

Comment:
Please set the actual fields from the database, so that data returned form the retrieve methods has all fields in it.